cmd/todo_list.go

```diff
@@ -33,8 +35,76 @@ var todoListCmd = &cobra.Command{
 		pager := NewPager(cmd.Flags())
 		defer pager.Close()
 
+		red := color.New(color.FgRed).SprintFunc()
```

Comment:
What about using another color (cyan?) for merged MRs instead of red?

Reply:
CYAN!?!?!?!?  I HATE CYAN.  Just joking, good idea.  I'll make this change in v2.

Reply:
Fixed.  Stupid cyan.